compiler-plugin/testData/api/annotationWithDefaultValues.kt

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Can you change this to 2022? Same as other occurrences.

Reply:
Done